### DISCUSSION TITLE
2024年6月29日购买的，邮箱还未收到邮件，软件也无法通过邮箱进行授权

### DISCUSSION THREAD

**Discussion by cqcao** [getlantern/lantern-forum-cn#655] [General]
1. 什么时候可以收到邮件
2. 软件如何绑定或者登录

**Comment by MaxMKMax** [comment]
@cqcao 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by cqcao** [reply]
我之前已经通过这个链接提交过信息的

**Comment by cqcao** [comment]
能重新发一下验证码吗？已经过期了&nbsp;&nbsp;你们的验证码一般有效期多久，或者你们什么时候能及时回复&nbsp;&nbsp;我就那时间找你们


---原始邮件---
发件人: &quot;Mike ***@***.***&gt;
发送时间: 2024年7月2日(周二) 凌晨0:16
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 2024年6月29日购买的，邮箱还未收到邮件，软件也无法通过邮箱进行授权 (Discussion #655)




 
@cqcao 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
 后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
 
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!
 
&mdash;
